pom.xml

```diff
@@ -12,14 +12,47 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.salesforce</groupId>
+    <groupId>com.salesforce.dockerfile-image-update</groupId>
     <artifactId>dockerfile-image-update-parent</artifactId>
     <packaging>pom</packaging>
     <version>1.0-SNAPSHOT</version>
+
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Parent pom ${project.groupId}:${project.artifactId}.</description>
+    <url>https://github.com/salesforce/dockerfile-image-update</url>
+    <organization>
+        <name>Salesforce</name>
+        <url>http://www.salesforce.com</url>
+    </organization>
+    <licenses>
+        <license>
+            <name>BSD 3-Clause</name>
```

Comment:
BSD 3-Clause License

Reply:
Done in 1f009e9d69b838eca260dd6e89810f4c47d98f1a